src/ensembl/src/content/app/custom-download/CustomDownload.tsx

```diff
@@ -1,24 +1,45 @@
-import React, { FunctionComponent } from 'react';
+import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
 import PreFilterPanel from 'src/content/app/custom-download/containers/pre-filters-panel/PreFilterPanel';
-import { getShowPreFilterPanel } from './state/customDownloadSelectors';
+import {
+  getShowPreFilterPanel,
+  getCustomDownloadActiveGenomeId
+} from './state/customDownloadSelectors';
 import { RootState } from 'src/store';
 import CustomDownloadHeader from './containers/header/CustomDownloadHeader';
 import CustomDownloadContent from './containers/content/CustomDownloadContent';
+import CustomDownloadAppBar from './containers/app-bar/CustomDownloadAppBar';
+import { setActiveGenomeId } from './state/customDownloadActions';
+import { CommittedItem } from 'src/content/app/species-selector/types/species-search';
+import { getCommittedSpecies } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
 
 type StateProps = {
-  showPreFiltersPanel: boolean;
+  shouldShowPreFilterPanel: boolean;
+  activeGenomeId: string | null;
+  committedItems: CommittedItem[];
 };
 
-type CustomDownloadProps = StateProps;
+type DispatchProps = {
+  setActiveGenomeId: (activeGenomeId: string) => void;
+};
+
+type CustomDownloadProps = StateProps & DispatchProps;
+
+const CustomDownload = (props: CustomDownloadProps) => {
+  useEffect(() => {
+    if (!props.activeGenomeId && props.committedItems.length) {
+      props.setActiveGenomeId(props.committedItems[0].genome_id);
+    }
+  }, []);
 
-const CustomDownload: FunctionComponent<CustomDownloadProps> = (
-  props: CustomDownloadProps
-) => {
+  if (!props.activeGenomeId) {
+    return null;
+  }
   return (
     <>
-      {props.showPreFiltersPanel && <PreFilterPanel />}
-      {!props.showPreFiltersPanel && (
+      <CustomDownloadAppBar onSpeciesSelect={props.setActiveGenomeId} />
+      {props.shouldShowPreFilterPanel && <PreFilterPanel />}
+      {!props.shouldShowPreFilterPanel && (
         <>
```

Comment:
No, there are two.
<CustomDownloadHeader /> 
<CustomDownloadContent />

Reply:
Ah I didn't look at it properly since both of them are in the same line. Can you put the second element in the next line so that it'll be clearer at first glance.